source/dub/compilers/dmd.d

```diff
@@ -86,7 +111,14 @@ config    /etc/dmd.conf
 		string[] arch_flags;
 		switch (arch_override) {
 			default: throw new Exception("Unsupported architecture: "~arch_override);
-			case "": break;
+			case "":
+				// Don't use Optlink by default on Windows
+				version (Windows) {
+					const is64bit = isWow64();
+					if (!is64bit.isNull)
```

Comment:
shouldn't this also check the value instead of just if it's null? `isWow64ProcessFp` could also return false.